### PR TITLE
Fix compiler warnings

### PR DESCRIPTION
**Pull request by joan-domingo**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -207,4 +207,12 @@ dependencies {
 
 }
 
+allprojects {
```
Comment (peter-tackage) [inline app/build.gradle]: Would it make more sense to have this in the top-level Gradle file?

Comment (peter-tackage) [inline app/build.gradle]: We only have one project at the moment, but I imagine that we would want to apply these checks globally.

Comment (joan-domingo) [inline app/build.gradle]: Indeed it would make more sense! I did not think it through

**Comment by joan-domingo** [comment]
I moved the gradle task to the top-level gradle file. Let me know if you have any more questions.

**Comment by peter-tackage** [comment]
Squash and merge when you are ready. Congrats on your first PR 🍾

**Comment by joan-domingo** [comment]
wohoo 🎆

**Pull request merged**